chainerx/_docs/routines.py

```diff
@@ -546,6 +546,56 @@ def _docs_linalg():
 .. seealso:: :func:`numpy.linalg.inv`
 """)
 
+    _docs.set_doc(
+        chainerx.linalg.svd,
+        """svd(a, full_matrices=True, compute_uv=True)
+Singular Value Decomposition.
+
+Factorizes the matrix ``a`` as ``u * np.diag(s) * v``, where ``u`` and
+``v`` are unitary and ``s`` is an one-dimensional array of ``a``'s
+singular values.
+
+Args:
+    a (~chainerx.ndarray): The input matrix with dimension ``(M, N)``.
+    full_matrices (bool): If True, it returns u and v with dimensions
+        ``(M, M)`` and ``(N, N)``. Otherwise, the dimensions of u and v
+        are respectively ``(M, K)`` and ``(K, N)``, where
+        ``K = min(M, N)``.
+    compute_uv (bool): If True, it only returns singular values.
```

Comment:
Sounds like the opposite.

Reply:
Changed to 
```
    compute_uv (bool): If True, only singular values are computed.
        u and v will be empty arrays.
```

Reply:
Fixed python binding such that when `compute_uv=False` only singular values array is returned.
Modified docs accordingly.